src/vs/base/browser/ui/iconLabel/iconLabel.ts

```diff
@@ -271,7 +335,7 @@ class LabelWithHighlights {
 
 				const name = dom.$('a.label-name', { id, 'data-icon-label-count': label.length, 'data-icon-label-index': i, 'role': 'treeitem' });
 				const highlightedLabel = new HighlightedLabel(dom.append(this.container, name), this.supportCodicons);
-				highlightedLabel.set(l, m, options?.title, options?.labelEscapeNewLines);
+				highlightedLabel.set(l, m, undefined, options?.labelEscapeNewLines);
```

Comment:
Same as above.

Reply:
Yes, if we don't pass undefined here then we can end up with a native tooltip too.